### PR TITLE
[4.0] Remove commented code

### PR DESCRIPTION
**Pull request by rdeutz**
Pull Request for Issue # .

### Summary of Changes
Cleanp of the scss files removeing commented code and unnecessary selectors 

### Testing Instructions
Code review

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 914955def80de719be434dba960843bc6ffd9149<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28035">issues.joomla.org/tracker/joomla-cms/28035</a>.</sub>

**Comment by brianteeman** [comment]
Really frustrating that I fixed all of this and more previously

**Comment by astridx** [comment]
I have tested this item :white_check_mark: successfully on 914955def80de719be434dba960843bc6ffd9149<br /><br />I had a look into the Code, I applied the patch and found no issue.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28035">issues.joomla.org/tracker/joomla-cms/28035</a>.</sub>

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 1]
```diff
@@ -99,13 +98,11 @@ joomla-toolbar-button {
         [class^='#{$fa-css-prefix}-'],
         [class*=' #{$fa-css-prefix}-'],
         span {
-         background: darken(theme-color("success"), 10%);
+          background: darken(theme-color("success"), 10%);
           color:rgba(255, 255, 255, 0.90);
```
Comment (HLeithner) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]:
```suggestion
          color: rgba(255, 255, 255, 0.90);
```

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 2]
```diff
@@ -99,13 +98,11 @@ joomla-toolbar-button {
...
-
-
         }
-               color:rgba(255, 255, 255, 0.90);
-               background-color: theme-color("success");
+        color:rgba(255, 255, 255, 0.90);
```
Comment (HLeithner) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]:
```suggestion
        color: rgba(255, 255, 255, 0.90);
```

File: administrator/templates/atum/scss/blocks/_form.scss [hunk 3]
```diff
@@ -2,10 +2,6 @@
 
 .form-control {
   max-width: $input-max-width;
-  /* background-color: var(--white-offset);
-  border:solid 2px var(--atum-bg-dark);
-
-   */
   background-color: $white;
   border:$input-border;
```
Comment (HLeithner) [inline administrator/templates/atum/scss/blocks/_form.scss]:
```suggestion
  border: $input-border;
```

**Review by HLeithner — CHANGES_REQUESTED** [review]

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28035">issues.joomla.org/tracker/joomla-cms/28035</a>.</sub>

**Comment by richard67** [comment]
@rdeutz I've allowed myself to commit @HLeithner suggested codestyle changes.

**Comment by richard67** [comment]
Thanks.

**Pull request merged**